packages/design-system/src/scalars/components/toggle/toggle-field.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { describe, it, vi } from "vitest";
+import { ToggleField } from "./toggle-field";
+
+describe("Toggle Component", () => {
+  const mockOnChange = vi.fn();
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
```

Comment:
If I'm not wrong, this is in the test setups so this is not needed here

Reply:
you are rigth, done